packages/graphql-yoga/__tests__/http-extensions.spec.ts

```diff
@@ -613,20 +615,43 @@ describe('Result Extensions', () => {
         throw(status: $status)
       }
     `
-    for (const statusCodeStr in STATUS_CODES) {
-      const status = Number(statusCodeStr)
-      const res = await yoga.fetch('http://yoga/graphql', {
-        method: 'POST',
-        headers: {
-          'content-type': 'application/json',
-        },
-        body: JSON.stringify({
-          query,
-          variables: { status },
-        }),
-      })
-      expect(res.status).toBe(status)
-      expect(res.statusText).toBe(STATUS_CODES[status])
-    }
+
+    it('should set status text by status code with yoga.fetch(...)', async () => {
+      for (const statusCodeStr in STATUS_CODES) {
+        const status = Number(statusCodeStr)
+        const res = await yoga.fetch('http://yoga/graphql', {
+          method: 'POST',
+          headers: {
+            'content-type': 'application/json',
+          },
+          body: JSON.stringify({
+            query,
+            variables: { status },
+          }),
+        })
+        expect(res.status).toBe(status)
+        expect(res.statusText).toBe(STATUS_CODES[status])
+      }
+    })
+
+    it('should set status text by status code with yoga(...) returning res', async () => {
```

Comment:
```suggestion
    it('should set status text by status code with yoga(...) returning res', async () => {
```
I think this test is extra. `yoga.fetch` does the same.

Reply:
I've removed the changes in this file